pokemongo_bot/inventory.py

```diff
@@ -1134,9 +1139,16 @@ def update_web_inventory(self):
         if not os.path.exists(web_inventory):
             self.init_inventory_outfile()
             
-        with open(web_inventory, "r") as infile:
-            json_inventory = json.load(infile)
-            infile.close()
+        try:
```

Comment:
If you mean the whole lot, including the try/except stuff... Not really. They're all slightly different.